tests/tests.xml

```diff
@@ -5,12 +5,15 @@
        <set name="unit-tests" feature="qtcontacts-sqlite-qt5">
            <description>Backend correctness automatic tests</description>
            <case manual="false" name="aggregation">
+               <step>rm -rf /home/nemo/.local/share/system/privileged/Contacts/qtcontacts-sqlite-test</step>
                <step>/opt/tests/qtcontacts-sqlite-qt5/tst_aggregation</step>
            </case>
            <case manual="false" name="contactmanager">
+               <step>rm -rf /home/nemo/.local/share/system/privileged/Contacts/qtcontacts-sqlite-test</step>
```

Comment:
I don't like the hardcoding of the nemo user here, but no doubt we've done the same thing elsewhere, I guess.
It'd be nice if we could somehow parametrise the user home path, though.